app/src/main/java/org/oppia/app/topic/questionplayer/QuestionPlayerViewModel.kt

```diff
@@ -42,24 +42,11 @@ class QuestionPlayerViewModel @Inject constructor() : ObservableViewModel() {
 
   fun getCanSubmitAnswer(): ObservableField<Boolean> = canSubmitAnswer
 
-  fun getPendingAnswer(
-    recyclerViewAssembler: StatePlayerRecyclerViewAssembler
-  ): UserAnswer {
-    return getPendingAnswerWithoutError(recyclerViewAssembler) ?: UserAnswer.getDefaultInstance()
-  }
-
-  private fun getPendingAnswerWithoutError(
-    recyclerViewAssembler: StatePlayerRecyclerViewAssembler
+  fun getPendingAnswerWithoutError(
+    answerHandler: InteractionAnswerHandler
   ): UserAnswer? {
-    val items = if (isSplitView.get() == true) {
-      rightItemList
-    } else {
-      itemList
-    }
```

Comment:
Is this case still being properly handled?

Reply:
It wasn't - I must have deleted this while fixing merge conflicts. Adding the functionality back now.